asus/zephyrus/ga402/default.nix

```diff
@@ -0,0 +1,20 @@
+{ ... }:
+
+{
+  imports = [
+    ../../../common/cpu/amd
+    ../../../common/cpu/amd/pstate.nix
+    ../../../common/gpu/amd
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/acpi_call.nix
+    ../../../common/pc/laptop/ssd
+  ];
+
+  services = {
+    asusd.enable = true;
```

Comment:
Should probably `mkDefault` here so people can opt out if they want.

Reply:
agreed

Reply:
fixed